app/views/layouts/application.html.erb

```diff
@@ -59,52 +59,55 @@
                 <div id="navbar" class="navbar-collapse collapse">
                     <ul class="nav navbar-nav navbar-right">
                         <% if session[:userinfo].present? %>
+
```

Comment:
<img width="1221" alt="screen shot 2017-05-14 at 6 08 55 am" src="https://cloud.githubusercontent.com/assets/7662875/26033586/7d2a2550-386c-11e7-895c-bc65c49447ba.png">
is this intended? have the word home in the logo? see picture